crates/ruff_python_parser/src/lexer.rs

```diff
@@ -1460,6 +1462,9 @@ impl std::fmt::Display for LexicalErrorType {
         match self {
             LexicalErrorType::StringError => write!(f, "Got unexpected string"),
             LexicalErrorType::FStringError(error) => write!(f, "f-string: {error}"),
+            LexicalErrorType::InvalidByteLiteral => {
+                write!(f, "bytes can only contain ASCII literal characters")
```

Comment:
```suggestion
                write!(f, "Bytes can only contain ASCII literal characters")
```
Nit. To be coherent with the other error messages.